src/style.css

```diff
@@ -0,0 +1,128 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+  padding: 0;
+
+  --border-color: #f3f3f3;
+  --dark: #060b35;
+  --gray: #616070;
+  --blue: #00acdc;
```

Comment:
better to use :root selector for css variables
https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties